applications/asset_tracker/src/cloud_codec/cloud_codec.h

```diff
@@ -48,6 +49,8 @@ enum cloud_channel {
 	CLOUD_CHANNEL_LTE_LINK_RSRP,
 	/** The descriptive DEVICE data indicating its status. */
 	CLOUD_CHANNEL_DEVICE_INFO,
+	/** The RBG IR light levels on the device. */
+	CLOUD_CHANNEL_LIGHT_SENSOR
```

Comment:
Next time, add a comma like the previous guy did, then the next patch will be small like this one.

But I wouldn't bother updating the PR just for this.